doc/releases/changelog-dev.md

```diff
@@ -6,11 +6,15 @@
 
 * A new method `process_density_matrix` has been added to the `ProbabilityMP` and `DensityMatrixMP` classes, allowing for more efficient handling of quantum density matrices, particularly with batch processing support. This method simplifies the calculation of probabilities from quantum states represented as density matrices.
   [(#5830)](https://github.com/PennyLaneAI/pennylane/pull/5830)
-
+  
 * The `qml.PrepSelPrep` template is added. The template implements a block-encoding of a linear 
   combination of unitaries.
   [(#5756)](https://github.com/PennyLaneAI/pennylane/pull/5756)
-  
+
+* A new method `to_mat` has been added to the `FermiWord` and `FermiSentence` class, which allows computing the matrix representation of a given Fermi operator.
+  [(#5920)](https://github.com/PennyLaneAI/pennylane/pull/5920)
+
+
```

Comment:
```suggestion
```

Reply:
Could you please clarify if there is a specific reason for removing the description of the current PR from the changelog?

Reply:
My suggestion was to remove the extra blank line not the description. We should keep the description.

Reply:
Thanks for the clarification!